optuna_dashboard/ts/action.ts

```diff
@@ -265,6 +285,44 @@ export const actionCreator = () => {
       })
   }
 
+  const tellTrial = (
+    studyId: number,
+    trialId: number,
+    state: TrialStateFinished,
+    values?: number[]
+  ): Promise<void> => {
+    const message =
+      values === undefined
+        ? `id=${trialId}, state=${state}`
+        : `id=${trialId}, state=${state}, values=${values}`
+    return tellTrialAPI(studyId, trialId, state, values)
+      .then(() => {
+        const index = studyDetails[studyId].trials.findIndex(
+          (t) => t.trial_id === trialId
+        )
+        if (index === -1) {
+          enqueueSnackbar(`Unexpected error happens. Please reload the page.`, {
+            variant: "error",
+          })
+          return
+        }
+        setTrialState(studyId, index, state, values?.map(Number))
```

Comment:
`values` has been already converted.

```suggestion
        setTrialState(studyId, index, state, values)
```

Reply:
Fixed.